polaris-react/src/components/Page/components/Header/Header.tsx

```diff
@@ -98,7 +98,7 @@ export function Header({
     !actionGroups.length;
 
   const breadcrumbMarkup =
-    breadcrumbs.length > 0 ? (
+    (Array.isArray(breadcrumbs) && breadcrumbs.length > 0) || breadcrumbs ? (
```

Comment:
This will still render a container if an empty array is passed in. You'll have to check for that like you did in the page component i.e.

```suggestion
    breadcrumbs != null &&
    Array.isArray(breadcrumbs) &&
    breadcrumbs.length > 0(
```

Reply:
Ooh i forgot this spot good catch!

Reply:
Oh this won't work though if breadcrumbs is an object and not an array since your check is all &&s.

It would need to be

```suggestion
    (Array.isArray(breadcrumbs) &&
    breadcrumbs.length > 0) ||
    (!Array.isArray(breadcrumbs) &&
    breadcrumbs)
```

Reply:
I think you're right, I was just copy pasting and did it wrong. I can test when the change has been made